contracts/contracts/eosio.system/src/delegate_bandwidth.cpp

```diff
@@ -434,22 +295,25 @@ namespace eosiosystem {
    }
 
    void system_contract::delegatebw( const name& from, const name& receiver,
-                                     const asset& stake_net_quantity,
-                                     const asset& stake_cpu_quantity, bool transfer )
+                                     const asset& stake_quantity,
+                                     bool transfer )
    {
       asset zero_asset( 0, core_symbol() );
+      const asset stake_net_quantity = stake_quantity / 2;
+      const asset stake_cpu_quantity = stake_quantity - stake_net_quantity;
```

Comment:
where do you set ram quota?

Reply:
we compute ram_bytes in
`changebw( from, receiver, stake_net_quantity, stake_cpu_quantity, transfer);`

Reply:
so, why don't we divide by 3 then?

Reply:
Because only the stake for CPU and NET used as a weight value. In general, we may change and set the stake to both NET and CPU without division. There is no difference.

Reply:
But if we would want to combine CPU and NET inside of **changebw** function it will require more changes in this function related to transfer 
```
auto transfer_amount = net_balance + cpu_balance;
         if ( 0 < transfer_amount.amount ) {
            token::transfer_action transfer_act{ token_account, { {source_stake_from, active_permission} } };
            transfer_act.send( source_stake_from, stake_account, asset(transfer_amount), "stake bandwidth" );
         }
```